src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -6,13 +6,13 @@ import { ProtocolVersion } from './data/dataTypes';
 import { getDuplicates } from '../../../../common';
 
 const RESERVED_LOCAL_HOST_IPS = {
-  ipv4: '127.0.0.0',
-  ipv6: '::1',
+  ipv4: ['127.0.0.0', '127.0.0.1'],
+  ipv6: ['::1', '::ffff:7f00:1'],
 };
 
 const RESERVED_CATCH_ALL_IPS = {
-  ipv4: '0.0.0.0',
-  ipv6: '0000::',
+  ipv4: ['0.0.0.0', '255.255.255.255'],
+  ipv6: ['0000::', '::ffff:ffff:ffff'],
```

Comment:
Sorry I hadn't noticed before., but I've noticed with this solution there is a problem due to the existing `compareIPV6Addresses` function.

From what I read, the loopback etc ipV6 addresses can be written in many different ways, and since with `compareIPV6Addresses` we're comparing the strings, we wouldn't detect all invalid addresses.

We're already using the dependency `ip-address` and it has functions for `isMultiCast` etc.

So I suggest to not use our own definition of ipV4 / ipv6 localhost / loopback / multicast etc as we were doing before, but to use the definitions in `ip-address`

Reply:
I'll just leave this here http://ip-address.js.org/#address6

Reply:
Added new validations for ipv6 addresses